src/client/eppo-client.ts

```diff
@@ -835,6 +838,83 @@ export default class EppoClient {
     throw err;
   }
 
+  private getAllAssignments(
+    subjectKey: string,
+    subjectAttributes: Attributes = {},
+  ): Record<string, PrecomputedFlag> {
+    const configDetails = this.getConfigDetails();
+    const flagKeys = this.getFlagKeys();
+    const flags: Record<string, PrecomputedFlag> = {};
+
+    // Evaluate all the enabled flags for the user
+    flagKeys.forEach((flagKey) => {
+      const flag = this.getFlag(flagKey);
+      if (flag === null || !flag.enabled) {
+        logger.info(`[Eppo SDK] No assigned variation. Flag is disabled: ${flagKey}`);
```

Comment:
Did you consider this at the `debug` level? I can see it being a bit noisy if users want to ignore it but they may want to see it if something is wrong (hence debug). Just a thought feel free to ignore if you feel this is defensible.

Reply:
ya, no need for info level